uninstall.sh

```diff
@@ -307,8 +292,10 @@ unregister_repo_for_autoupdate() {
 #   0 on success, 1 on failure
 ############################################################
 uninstall_from_current_repository() {
-    if ! is_running_in_git_repo_root; then
-        echo "The current directory ($(pwd)) does not seem to be the root of a Git repository!" >&2
+    CURRENT_GIT_DIR=$(cd "$(git rev-parse --git-common-dir 2>/dev/null)" && pwd)
+
+    if ! is_git_repo "$(pwd)" || [ ! -d "$CURRENT_GIT_DIR" ]; then
```

Comment:
Wouldn't `! is_git_repo "$CURRENT_GIT_DIR"` be enough here?

Reply:
Oh the git dir will point to a `.git` directory normally?
In that case, it feels like the `git rev-parse` check should be inside the `is_git_repo` function.

Reply:
Correct. if we are inside a git repo, he directory should be valid! :-)

Reply:
Note: the `if ! is_git_repo "$(pwd)" ` is only to check if `uninstall_from_current_repository` is really run from inside a git repository, then also `CURRENT_GIT_DIR` is set and valid. I will change that

Reply:
See latest review commits.